### DISCUSSION TITLE
Is there any npm equivalent to "yarn corde init"?

### DISCUSSION THREAD

**Discussion by itsariadust** [cordejs/corde#1318] [Q&A] [answered]
So I've been wanting to use this package for a while and I've been scouring through the documentation but I don't see any equivalent to running `yarn corde init`. I don't really want to switch package managers or even use two at the same time (yarn already warned me about it before). Is there anything equivalent to it for npm or do I have to force myself to use yarn?

**Comment by lucasgmagalhaes** [comment, marked as answer]
Hi @itsariadust! You can install corde globally and then call 

```bash
corde init <fileExtension>
```

Or install locally and call with `npx`

```bash
npx corde init <fileExtension>
```

Sorry for missing documentation about it. I gonna put it. Feel free to add it by yourself if you want!
